onyxia-api/src/main/java/fr/insee/onyxia/api/dao/universe/CatalogRefresher.java

```diff
@@ -52,6 +53,12 @@ private void refresh() {
         }
     }
 
+    @PostConstruct
```

Comment:
Now that you chose the `healthcheck` solution, this `@PostConstruct` method is not suitable anymore as it would do double refresh and refresh before accepting network connections